README.md

```diff
@@ -28,9 +28,9 @@ A website that lets you know where to watch a movie built on [Next.js](https://n
 </p>
 
 ## Deployment
-- Use the deploy button below and follow the instructions
-- After the project is successfully deployed, verify that your Meilisearch credentials have been automatically added to Vercel
-- Then, proceed to redeploy the project in Vercel's dashboard so that it can access the newly set Meilisearch credentials
+1. Click on the 'Deploy' button below and follow the instructions
+2. The deployment may fail. In such a case, please verify that your Meilisearch credentials have been automatically added to Vercel
```

Comment:
> The deployment may fail

The deployment _will_ fail, right?

I suggest including a comment explaining why, eg. "The deployment will fail because Meilisearch has not finished initializing"